jest.config.js

```diff
@@ -107,15 +107,15 @@ module.exports = {
   // rootDir: null,
 
   // A list of paths to directories that Jest should use to search for files in
-  // roots: [
-  //   "<rootDir>"
-  // ],
+  roots: [
+    "<rootDir>/test/unit"
```

Comment:
We don't want to look in other folders for JS tests - specifically the new `setupTests.js` file shouldn't be run as a unit test with Jest.